packages/apollo-engine-reporting-protobuf/src/reports.proto

```diff
@@ -109,18 +119,65 @@ message Trace {
 		reserved 4;
 	}
 
+	// FIXME: document all new fields
+	message QueryPlanNode {
+		message SequenceNode {
+			repeated QueryPlanNode nodes = 1;
+		}
+		message ParallelNode {
+			repeated QueryPlanNode nodes = 1;
+		}
+		message FetchNode {
+			string serviceName = 1;
+			string operation = 2;
+			// FIXME: do we want selectionSet to be structured instead of a string?
+			// FIXME: do we need variableUsages?
+			// FIXME: do we need requires?
+
+			// This Trace only contains start_time, end_time, duration_ns, and root;
+			// all timings were calculated on the federated service.
```

Comment:
Be explicit somewhere that all fields on fetchnode are measured on gateway (as an overall statement).